fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/physical/PhysicalOperator.java

```diff
@@ -5,14 +5,17 @@
 import com.starrocks.sql.optimizer.base.ColumnRefSet;
 import com.starrocks.sql.optimizer.base.DistributionSpec;
 import com.starrocks.sql.optimizer.base.OrderSpec;
+import com.starrocks.sql.optimizer.base.PhysicalPropertySet;
 import com.starrocks.sql.optimizer.operator.Operator;
 import com.starrocks.sql.optimizer.operator.OperatorType;
 
+import java.util.List;
 import java.util.Set;
 
 public abstract class PhysicalOperator extends Operator {
     protected OrderSpec orderSpec;
     protected DistributionSpec distributionSpec;
+    private List<PhysicalPropertySet> requiredProperties;
```

Comment:
I suggest move it to OptExpression, because it's only used after extact plan phase, it's will make operator simple, just advise

Reply:
done